.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -33,58 +33,20 @@ body:
       render: "true"
     validations:
       required: true
-  - type: input
-    attributes:
-      label: Active renderer
-      description: For graphical and input-related issues. You can find these in the About tab in the mainmenu.
-      placeholder: "Example: OpenGL 4.6.0"
-    validations:
-      required: false
-  - type: input
-    attributes:
-      label: Irrlicht device
-      description: 
-      placeholder: "Example: X11"
-    validations:
-      required: false
-  - type: input
-    attributes:
-      label: Operating system and version
-      description: It is recommended to upgrade your operating system to see if the problem still exists.
-      placeholder: "Example: Ubuntu 22.04"
-    validations:
-      required: true
-  - type: input
-    attributes:
-      label: CPU model
-      description: Usually found in system settings. 
-      placeholder: "Example: Intel i5-2410M (4) @ 2.900GHz"
-    validations:
-      required: false
-  - type: markdown
-    attributes:
-      value: The GPU model and OpenGL version can be omitted if the bug is not a graphical issue.
-  - type: input
-    attributes:
-      label: GPU model
-      description: Usually found in system settings. 
-      placeholder: "Example: NVIDA GeForce RTX 4090"
-    validations:
-      required: false
-  - type: input
+  - type: textarea
     attributes:
-      label: OpenGL version
-      placeholder: "Example: 4.6"
+      label: System Information
+      description: Operating System? CPU model? GPU? OpenGL version?
```

Comment:
Please keep the instructions to get "Active renderer" and "Irrlicht device" fields from credits tab.
This is important now that we're switching to SDL, and might also switch to opengl 3.

Reply:
Then let's add a short blurb about how to find that information.

I recently opened an issue and I personally found it daunting and I work on MT. :(

Reply:
This is part of the version information

Reply:
> Then let's add a short blurb about how to find that information.

>The current form doesn't even say what an Irrlicht device is or how to find it out.

It says:

>For graphical and input-related issues. You can find these in the About tab in the mainmenu.

This used to be above active renderer and irrlicht device. It was incorrectly moved to active renderer only on the change to non-text-only issue form.


---

>not relevant to all issues

That's because you can usually conclude it from the OS and minetest version. We can maybe write that the information needs only to be supplied if non-default build options were used.

---

>This is included as part of the version information if you do --version.

(Non-trivial to add in `--version`, because as of now it doesn't create an irrlicht device.)

You also need to consider that issues may be opened from the current stable version, i.e. 5.8.0, where there's no export button or similar for this.